### ISSUE TITLE
[haxedevelop] Trim Trailing Whitespace not working

### ISSUE THREAD

**Issue by ncannasse** [fdorg/flashdevelop#1106]
When activating "Trim Trailing Whitespace" in Formatting options, this does not make any difference while it was working correctly with FD 5.0.1.3

**Comment by Neverbirth** [comment]
Strange, it works completely fine here, tested with FD and HD distros using both 32 and 64bits builds, and I don't see any suspicious code.

**Comment by Gama11** [comment]
There's two different "Trim Trailing Whitespace" options, one in `FlashDevelop` and one in `CodeFormatter`. Maybe that has to do with it? The latter wouldn't work work Haxe I guess.

**Comment by Neverbirth** [comment]
Yes, but the one for CodeFormatter is not under "Formatting", so I'd say the first one is used.

**Comment by ncannasse** [comment]
That was it ! I had CodeFormatter enabled, but it needed HaxeDevelop option.

Actually I would like to report two related bugs :)

1) when you filter with "Trim" , it will keep both HaxeDevelop and CodeFormatter sections, but clicking on HaxeDevelop will show nothing...

2) there should not be two Trim Whitespaces options in the first place
